apps/cowswap-frontend/src/common/updaters/orders/ExpiredOrdersUpdater.ts

```diff
@@ -37,12 +35,8 @@ export function ExpiredOrdersUpdater(): null {
         // - Owned by the current connected account
         // - Created in the last 5 min, no further
         // - Not yet refunded
-        const pending = expiredRef.current.filter(({ owner, creationTime: creationTimeString, refundHash }) => {
```

Comment:
Can you update the comment to reflect the changes?

Reply:
Done